src/main/include/subsystems/Vision.hpp

```diff
@@ -28,15 +28,20 @@ class Vision : public frc2::SubsystemBase {
 
                                 735.0038676061282,
                             0.0, 666.4393853583332, 0.0, 734.0230911175827,
-                            388.3489253256961, 0.0, 0.0, 1.0);
+                            388.3489253256961, 0.0, 0.0, 1.0
```

Comment:
Could you add some comments documenting which constants are for which camera (on your Mac, the global shutter one, etc.).

Reply:
Done